packages/accounts-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.3]
+### Changed
+- Bump dependency on `@metamask/keyring-controller` to ^8.0.3
+- Add `test:clean` build script that clears jest cache ([#1714](https://github.com/MetaMask/core/pull/1714))
```

Comment:
I think this was alluded to, but we tend to gear the changelog for a package toward consumers. Since this seems to be a development-only change, I believe we can omit this entry.

``` suggestion
```